README.md

```diff
@@ -15,8 +15,17 @@ To register an extension, open a new PR with a new extension directory under the
  * [Storage Transformers](./storage-transformers/README.md)
 
 Each extension MUST have a `README.md` file that describes the extension and its metadata specification.
```

Comment:
```suggestion
Each extension MUST have a `README.md` file that describes the extension.
```

Suggesting a slight modification here because this could be read as if the specification must be housed here.